### DISCUSSION TITLE
I cannot see the report folder

### DISCUSSION THREAD

**Discussion by jitsingh007** [detekt/detekt#5262] [Q&A] [answered]
Hello There,
After integrating the detekt plugins in android project and can't see report folder in build folder. Folder generated are .transform/ generated, intermediate, outputs, roomSchemas, & tmp

**Comment by schalkms** [comment, marked as answer]
How does your configuration look like?
Did detekt actually run over project?

Unfortunately, without giving more information, it becomes hard to pinpoint the problem.

**Reply by jitsingh007** [reply]
Hi @schalkms, I'm integrating detekt tool in android project. But facing some issue in integration i.e., not able to generate the report with detekt plugin. According to my understanding the detekt plugin should be included in project level and module level; but what if I want to generate a combined report of project and in project I have 30 submodules. My question: do I need to include detekt plugins in all 30 submodules or I can include in project build.gradle file and get a combined report.

**Reply by schalkms** [reply]
Detekt provides an application sample for merging and combining reports. You can find it [here](https://detekt.dev/docs/introduction/reporting#merging-reports).

This section should answer your question.
